testData/preferences/mary.json

```diff
@@ -4,10 +4,10 @@
             "gpii-default": {
                 "name": "Default preferences",
                 "preferences": {
-                    "http://registry.gpii.net/common/onScreenKeyboardEnabled": true,
+                    "http://registry.gpii.net/common/onscreenKeyboard": true,
```

Comment:
yes! That sounds good to me - though we should make sure to update the spreadsheet as well!